pkg/bpf/map_linux_test.go

```diff
@@ -110,9 +109,10 @@ var (
 	maxEntries = 16
 )
 
-func mapsEqual(a, b *Map) bool {
-	return a.name == b.name &&
-		reflect.DeepEqual(a.spec, b.spec)
+func mapsEqual(t *testing.T, a, b *Map) {
```

Comment:
Lets make this signature more similar to the testing functions we are calling for readability:
```suggestion
func mapsEqual(t *testing.T, expected, actual *Map) {
```

As a reference the signatur of require.Equal:
```func Equal(t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{})```

Reply:
Done!